### DISCUSSION TITLE
Error in ww3.gridded.nc result

### DISCUSSION THREAD

**Discussion by ZHHisme** [NOAA-EMC/WW3#663] [Q&A]
My simulation area is as follows (pic1). bot2.txt、mask.txt(I have attached) are my input datas about my simulation area's topography.
However why the WW3's result's showed area(pic2) is quite different form my original area????
pic1::::
![matlabpic](https://user-images.githubusercontent.com/103922752/163777003-f8ce5fa1-99d7-4aeb-8653-6e059799e693.png)

pic2:::
![屏幕截图 2022-04-18 155150](https://user-images.githubusercontent.com/103922752/163775926-a1a1381c-cd45-4a0a-b35d-f88ceaaa1083.png)

[bot2.txt](https://github.com/NOAA-EMC/WW3/files/8504394/bot2.txt)
[mask.txt](https://github.com/NOAA-EMC/WW3/files/8504396/mask.txt)

__________________________Here is my  ww3_grid.inp________________________________________________
$
   1.1 0.04118 25 24 0.
$
   F T T T F T
   450.  450.  450.   15.
$
  &MISC CICE0 = 0.25, CICEN = 0.75, FLAGTR = 0 / 
  &FLX3 CDMAX = 3.5E-3 , CTYPE = 0 /
  &OUTS E3D = 1 /
$
END OF NAMELISTS
$
      'RECT'  T  'NONE'
        1901	   1261                  
     3 	  3	 60.00 
     84.975 	   -6.025 	  1.00
$
  -0.1 0.25  40  -1. 3 1 '(....)' 'NAME' 'bot2.txt'
$
   60  1 1 '(....)' 'NAME' 'mask.txt'
$
$
   0. 0. 0. 0.  0

**Comment by aliabdolali** [comment]
@ZHHisme It seems your depth should be multiplied by -1 or you might need to use +1 instead of -1 to read your depth file correctly. Look at the manual and [model/inp/ww3_grid.inp](https://github.com/NOAA-EMC/WW3/blob/develop/model/inp/ww3_grid.inp) 
`-0.1 0.25 40 1. 3 1 '(....)' 'NAME' 'bot2.txt'`

**Reply by ZHHisme** [reply]
I get it, thanks a lot!
